src/general-components/Session/Session.tsx

```diff
@@ -71,6 +71,21 @@ class Session {
     static removeTokens = () => {
         Session.token.delete();
         Session.refreshToken.delete();
+
+        // Darkmode
+        localStorage.removeItem("darkmode");
+
+        let settings = SettingsList.FromArray([{
+            name: "Dark Mode",
+            value: "false",
+            id: 1,
+            type: "",
+            extras: "",
+            description: "",
+            exists: true,
+            default: "false"
+        }]);
+        DarkModeChanger(settings, settings);
```

Comment:
In my oppinion, this code should be placed inside the DarkModeChange file. I think it should be possible to register a user change listener and make these changes if the user is null, to decouple the DarkMode code from the session code.

Reply:
Applied changes.